doc/changelog.md

```diff
@@ -67,6 +67,19 @@
 
   ```
 
+* Support for including a list of previously excluded modules
+  for Autograph conversion.
+  [(#725)](https://github.com/PennyLaneAI/catalyst/pull/725)
```

Comment:
It may be good to state in the changelog why you would want to do this (briefly).

Reply:
Done